learn4haskell.cabal

```diff
@@ -22,7 +22,7 @@ source-repository head
   location:            https://github.com/kowainik/learn4haskell.git
 
 common common-options
-  build-depends:       base ^>= 4.14.0.0
+  build-depends:       base ^>= 4.13.0.0
```

Comment:
That means, that locally you have an older version of GHC. Could you check:

```shell
ghc --version
```

Also, could you please share if you used the instructions in the README to install all necessary?

Reply:
Also, we made the whole infrastructure work with GHC-8.10, including CI. So, now the CI won't build the project.

But I guess, we can resolve the root of the problem.

Reply:
Ah, you are right.  I had to restart the setup and I must have missed that step the second time.  Everything is working fine now locally.  Thank you!

Reply:
I think I spoke too soon.  There is a problem with the Haskell extension for Visual Studio Code and it has this error message:

> The Haskell (learn4haskell) server crashed 5 times in the last 3 minutes. The server will not be restarted.

I removed my installation and started everything again, following the README.  `ghc` is at version 8.10.2 and `cabal` is at version 3.2.0.0.

Any ideas here?

Reply:
> The Haskell (learn4haskell) server crashed 5 times in the last 3 minutes. The server will not be restarted.

Is this a full error message, probably you have some more logs?

Sorry, that it doesn't work for you. 😞  Maybe if it would continue crashing you could open the issue in the plugin repo and ask them for advice? I think they would be able to help better than me here, as I am not a user of VSCode, but I did check it on my laptop, and I haven't those problems.

Reply:
I ended up trying stack which is working for me.  Thank you for the help :)